modules/lang-painless/src/main/java/org/elasticsearch/painless/ir/FunctionNode.java

```diff
@@ -136,6 +185,34 @@ protected void write(ClassWriter classWriter, MethodWriter methodWriter, Globals
         methodWriter = classWriter.newMethodWriter(access, method);
         methodWriter.visitCode();
 
+        // TODO: do not specialize for execute
+        Label startTry = new Label();
```

Comment:
Can you do something like:
```
try {

} catch (XXX) {

}
```

Reply:
It's hard to understand what's going here without some clarity.

Reply:
Just a sign-post here would be fine.  "Start handling try"

Reply:
Added several comments to clarify what's happening in this code